_events/2023/2023-06-education-training-talk.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Machine Learning for your Research
+subtitle: US-RSE Education & Training Seminar Series
+expires: 2023-06-22
+event_date: "June 22, 2023"
+layout: event
+duration: 120
+repeated: false
+category: education-training
+time:
+    - - start: 2023-06-22T18:00:00Z
+        end: 2023-06-22T20:00:00Z
+---
+
+## US-RSE Education & Training Seminar Series
+
+US-RSE periodically presents technical talks and tutorials related to Education & Training for RSEs.
+
+## Event
+
+The next technical talk of the US-RSE Education & Training Seminar Series will feature Savannah Thais who will speak about "Machine Learning for your Research."
+This event will take place **Thursday, June 22nd at 2 PM ET, 1 PM CT, 12 PM MT, 11 AM PT**
```

Comment:
Since the event is 2 hours should we say 2-4pm in the text?

Reply:
We say at the top that it is 2 hours, but we can add it here if you think it would be helpful. Could you sugget a way to phrase it? Do we want to say 2-4 PM ET, 1-3 PM CT, 12-2PM MT, 11AM-1PM PT?

Reply:
Updated.